boot/term.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+# Import device specific settings like the port
+source /lib/live/mount/medium/config.sh
```

Comment:
Put this after the SERVER line so the config file can override the server if it needs to (e.g. local testing)